website/blog/2018-11-21-upgrade-node-version-circleci.md

```diff
@@ -0,0 +1,36 @@
+---
+title:     "How to upgrade node on CircleCI machine executor"
+author:    Manuel Mazzuola
+authorURL: https://keybase.io/originof
+---
+
+CircleCI offers 3 different [executor types](https://circleci.com/docs/2.0/executor-types/), machine, Docker and MacOS. On Linux the best choice would be the docker executor but sometimes you need full control over job environment so you must opt for the machine executor.
```

Comment:
... but if you need full control over the job environment you must opt for the machine executor.